spec/controllers/users_controller_spec.rb

```diff
@@ -5,15 +5,15 @@
     it 'redirects and displays the flash message if no user is present' do
       delete :destroy
 
-      expect(response).to redirect_to(root_url)
+      expect(response).to redirect_to "http://#{Figaro.env.domain_name}/"
```

Comment:
Apparently this is due to returning a `_url` instead of `_path` in the decorated session. Switching to `_path` allows to use the URL helpers in the tests. I'll make the change.